syncano/models/manager.py

```diff
@@ -711,6 +711,28 @@ def exclude(self, *args):
         self.endpoint = 'list'
         return self
 
+    def ordering(self, order=None):
+        pass
```

Comment:
I think we should raise an exception here. Saying that user should use order_by. Silently failing may be tough to debug.